# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvi/no-machine-setting.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile~	Thu Oct 28 09:15:53 1999
+++ ./Makefile	Thu Oct 28 09:18:40 1999
@@ -6,7 +6,7 @@
 #
 
 CC		= gcc
-CFLAGS	= -O6 -fomit-frame-pointer -m486 -s -pipe
+CFLAGS	= -O6 -fomit-frame-pointer -s -pipe
 
 BINDIR		= /usr/bin
 MANPREFIX	= n
